libraries/adaptive-expressions/src/builtinFunctions/titleCase.ts

```diff
@@ -22,17 +24,24 @@ export class TitleCase extends StringTransformEvaluator {
     }
 
     /**
+     * @param args
+     * @param options
      * @private
      */
-    private static evaluator(args: any[]): string {
-        const inputStr = String(InternalFunctionUtils.parseStringOrUndefined(args[0])).toLowerCase();
-        if (inputStr === '') {
-            return inputStr;
-        } else {
-            return inputStr.replace(
-                /\w\S*/g,
-                (txt): string => txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase()
-            );
+    private static evaluator(args: unknown[], options: Options): string {
+        let locale = options.locale ? options.locale : Intl.DateTimeFormat().resolvedOptions().locale;
+        locale = FunctionUtils.determineLocale(args, 2, locale);
+        const firstArg = args[0];
+        if (typeof firstArg === 'string' || firstArg === undefined) {
+            const inputStr = (InternalFunctionUtils.parseStringOrUndefined(firstArg) as any).toLocaleLowerCase(locale);
```

Comment:
Why the `as any`?

Reply:
tsc compiler throws unexpected error, if I remove as any here. 
```
src/builtinFunctions/titleCase.ts:31:103 - error TS2554: Expected 0 arguments, but got 1.
           const inputStr = InternalFunctionUtils.parseStringOrUndefined(firstArg).toLocaleLowerCase(locale);
```